From a579d261336e76b93b93c1f6db5586edf4b5db0e Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 8 Feb 2021 20:47:12 +0000
Subject: IWYU: add missing include for std::unique_ptr

Bug: None
Change-Id: I86445b69402d31adf4f616eeb8ecea20ba97fe67
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2679524
Reviewed-by: Nico Weber <thakis@chromium.org>
Commit-Queue: Nico Weber <thakis@chromium.org>
Cr-Commit-Position: refs/heads/master@{#851872}
---
 ui/gfx/rendering_stage_scheduler.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gfx/rendering_stage_scheduler.h b/ui/gfx/rendering_stage_scheduler.h
index 701f62b71c7b..3c00f17b4319 100644
--- a/ui/gfx/rendering_stage_scheduler.h
+++ b/ui/gfx/rendering_stage_scheduler.h
@@ -5,6 +5,7 @@
 #ifndef UI_GFX_RENDERING_STAGE_SCHEDULER_H_
 #define UI_GFX_RENDERING_STAGE_SCHEDULER_H_
 
+#include <memory>
 #include <vector>
 
 #include "base/threading/platform_thread.h"
-- 
2.25.1

